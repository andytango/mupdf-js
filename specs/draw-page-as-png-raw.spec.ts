import {
  forExamplePages,
  loadExampleFile,
  TEST_DPI,
} from "./helpers/example-file";
import {hash} from "./helpers/hash";

describe("drawPageAsPNGRaw", () => {
  forExamplePages("should return a png file as Uint8Array", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(hash(Buffer.from(mupdf.drawPageAsPNGRaw(doc, page, TEST_DPI)))).toMatchSnapshot();
  });
});
