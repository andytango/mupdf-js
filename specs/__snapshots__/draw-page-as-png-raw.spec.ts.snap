// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drawPageAsPNGRaw Page [1 / 4] - should return a png file as Uint8Array 1`] = `"54503f92cfcf175478d0bbd081fa0258"`;

exports[`drawPageAsPNGRaw Page [2 / 4] - should return a png file as Uint8Array 1`] = `"a0ec0d2f8091065021e56468f25a6f97"`;

exports[`drawPageAsPNGRaw Page [3 / 4] - should return a png file as Uint8Array 1`] = `"e81a49d1900df3408f0972800413fa4f"`;

exports[`drawPageAsPNGRaw Page [4 / 4] - should return a png file as Uint8Array 1`] = `"1d034eab46638f0f12dd487c574da774"`;
